com.visualstudio.code.yaml

```diff
@@ -42,6 +42,16 @@ modules:
 
   - shared-modules/libsecret/libsecret.json
 
+  - name: zypak
```

Comment:
Zypak is included by default when using

```yaml
base: org.electronjs.Electron2.BaseApp
```

https://docs.flatpak.org/en/latest/electron.html#the-electron-base-app

Reply:
Thank you! I am not the most familiar with Flatpaks, so this was very helpful. Added in [`345fdbd`](https://github.com/bytzo/com.visualstudio.code/commit/345fdbd4a663d78abe736e016174f1f512a277f1).